packages/autocertifier/tsconfig.jest.json

```diff
@@ -0,0 +1,19 @@
+{
+    "extends": "../../tsconfig.jest.json",
+    "compilerOptions": {
+        "outDir": "dist",
+        "noImplicitOverride": false
+    },
+    "include": [
+        "src/**/*",
+        "test/**/*"
+    ],
+    "exclude": [
+        "src/connection/WebRTC/BrowserWebRtcConnection.ts"
+    ],  
```

Comment:
Remove the exclude as the file doesn't exist in this package? (the same reference is in multiple files in `autocertificer` and `autocertifier-client` packages)